src/command_modules/azure-cli-cosmosdb/azure/cli/command_modules/cosmosdb/_params.py

```diff
@@ -52,3 +52,6 @@ def load_arguments(self, _):
         c.argument('partition_key_path', help='Partition Key Path, e.g., \'/properties/name\'')
         c.argument('indexing_policy', type=shell_safe_json_parse, completer=FilesCompleter(), help='Indexing Policy, you can enter it as a string or as a file, e.g., --indexing-policy @policy-file.json)')
         c.argument('default_ttl', type=int, help='Default TTL')
+
+    with self.argument_context('cosmosdb database') as c:
+        c.argument('throughput', type=int, help='Offer Throughput')
```

Comment:
What are the units for this?  They should be called out in the help text.

Reply:
added units in latest iteration